scripts/pre_commit_linter.py

```diff
@@ -548,15 +542,15 @@ def _check_bad_patterns(all_files):
                         filename, BAD_PATTERNS[pattern]['message'])
                     total_error_count += 1
             if filename.endswith('.js'):
-                for pattern in BAD_PATTERNS_JS:
-                    if filename not in (
-                            BAD_PATTERNS_JS[pattern]['excluded_files']):
-                        if pattern in content:
-                            failed = True
-                            print '%s --> %s' % (
-                                filename,
-                                BAD_PATTERNS_JS[pattern]['message'])
-                            total_error_count += 1
+                # for pattern in BAD_PATTERNS_JS:
```

Comment:
OK to just delete all this stuff if it's not needed. (We have the git repo around for posterity if we ever need to refer back to it in the future.)